### PR TITLE
[AAE-5145] Add enum for content node selector actions

### PR DESCRIPTION
**Pull request by TomikaArome**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [x] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

https://alfresco.atlassian.net/browse/AAE-5145
The actions are defined as simple strings.

**What is the new behaviour?**

The actions are defined using a new enumerations with values set to strings.

**Does this PR introduce a breaking change?** (check one with "x")

> - [x] Yes
> - [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications:

Both of these PRs will have to be merged:
ACA: https://github.com/Alfresco/alfresco-content-app/pull/2149
ADW: https://github.com/Alfresco/alfresco-digital-workspace-app/pull/1246

**Other information**:

### CODE REVIEW

**Review by arditdomi — COMMENTED** [review]
Should we keep the Enum suffix? I suggest we call it just NodeActions

**Comment by TomikaArome** [comment]
> Should we keep the Enum suffix? I suggest we call it just NodeActions

I was wondering this myself. I've seen the Enum suffix used elsewhere and many other things use suffixes (like components and services), but I can see the case for not using it for enums.

**Comment by arditdomi** [comment]
> > Should we keep the Enum suffix? I suggest we call it just NodeActions
> 
> I was wondering this myself. I've seen the Enum suffix used elsewhere and many other things use suffixes (like components and services), but I can see the case for not using it for enums.

The only reason I don't like it is imagine having a string variable and naming it with the string suffix. nodeIdString, or hasMoreItemsBoolean. Sounds strange but I will approve it and leave it up to you 🙂 It doesn't really bother me in this case

**Review by arditdomi — APPROVED** [review]

**Comment by TomikaArome** [comment]
> The only reason I don't like it is imagine having a string variable and naming it with the string suffix. nodeIdString, or hasMoreItemsBoolean. Sounds strange but I will approve it and leave it up to you 🙂 It doesn't really bother me in this case

That's true, and I think you make a valid point there. I will change it. Better now than later.

File: lib/content-services/src/lib/document-list/models/node-action.enum.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+/*!
+ * @license
+ * Copyright 2019 Alfresco Software, Ltd.
```
Comment (andrea-ligios) [inline lib/content-services/src/lib/document-list/models/node-action.enum.ts:3]: It's not 2019 anymore :)

**Review by arditdomi — APPROVED** [review]

**Review by andrea-ligios — APPROVED** [review]
Net the copyright notice, LGTM

File: lib/content-services/src/lib/document-list/models/node-action.enum.ts (hunk 1 above)
Comment (TomikaArome) [reply lib/content-services/src/lib/document-list/models/node-action.enum.ts:3]: I remember putting 2021 once and the linting or something like that failed.

**Review by eromano — APPROVED** [review]

**Pull request merged**